4.9/ga/ci-pipeline.md

```diff
@@ -161,6 +161,28 @@ oc import-image redhat-marketplace-index \
   --all
 ```
 
+Due to the excessive number of tags present on these images, the command may fail with an error `Unable to connect to the server: EOF`. To work around this issue, import only the image which corresponds to the major and minor version of your OpenShift cluster.
```

Comment:
I'd just have this replace the existing section, and say "the `--all` flag will fail with the following error..."